src/background.js

```diff
@@ -1,7 +1,7 @@
 const toolsPorts = new Map()
 
 chrome.runtime.onConnect.addListener(port => {
-  if (port.sender.url == chrome.runtime.getURL('/devtools/panel.html')) {
+  if (port.sender.url.replace('//devtools', '/devtools') == chrome.runtime.getURL('/devtools/panel.html')) {
```

Comment:
```suggestion
  if (port.sender.url.replaceAll('//', '/') == chrome.runtime.getURL('/devtools/panel.html')) {
```

And what do you think about adding `toLowerCase()`?